test/test.cpp

```diff
@@ -144,9 +144,7 @@ class VideoParserClient : public VkParserVideoDecodeClient {
 
     bool UpdatePictureParameters(VkPictureParameters* params, VkSharedBaseObj<VkParserVideoRefCountBase>& shared, uint64_t count) final
     {
-        // fprintf(stdout, "%s: %" PRIu64 "\n", __FUNCTION__, count);
         fprintf(stdout, "[%lu] %s\n", syscall(SYS_gettid), __FUNCTION__);
-        shared = PictureParameterSet::create();
```

Comment:
AFAIK, the shared object is created by the parser library and returned to the application which will use it to store custom data.

Reply:
I was not sure if it was necessary indeed. My guess was the test application does not need this shared picture. Let me double check

Reply:
Should be better now :)